pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/SchemaBuilderTest.java

```diff
@@ -23,10 +23,7 @@
 
 import org.apache.avro.reflect.Nullable;
 import org.apache.pulsar.client.api.Schema;
-import org.apache.pulsar.client.api.schema.GenericRecord;
-import org.apache.pulsar.client.api.schema.GenericSchema;
-import org.apache.pulsar.client.api.schema.RecordSchemaBuilder;
-import org.apache.pulsar.client.api.schema.SchemaBuilder;
+import org.apache.pulsar.client.api.schema.*;
```

Comment:
nit: We should avoid this "*" kind import

Reply:
OK,I will change it